__tests__/fixtures/bundles/MockMeasureBundle.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
nitpick, but I don't think this warrants its own file since it's so small. You can probably just make this a `const MOCK_MEASURE_BUNDLE` at the top of the test file where you use it